src/core/dev/hw_queue_tx.cpp

```diff
@@ -2041,6 +2042,22 @@ void hw_queue_tx::clear_notification()
     }
 }
 
+doca_error_t hw_queue_tx::submit_txq_task(doca_task *task)
+{
+    // be default - don't decide to flush -
+    // but let doca decide to do so if they think it's the right thing to do
+    doca_task_submit_flag submit_flags = DOCA_TASK_SUBMIT_FLAG_OPTIMIZE_REPORTS;
+
+    if (is_completion_need()) {
+        submit_flags = DOCA_TASK_SUBMIT_FLAG_FLUSH;
```

Comment:
We should always use this flag. Currently XLIO does not support doorbell batching.

Reply:
fixed